readthedocs/templates/profiles/base_profile_edit.html

```diff
@@ -49,6 +49,7 @@ <h2>
           <li class="{% block profile-admin-social-accounts %}{% endblock %}"><a href="{% url 'socialaccount_connections' %}">{% trans "Connected Services" %}</a></li>
           <li class="{% block profile-admin-change-password %}{% endblock %}"><a href="{% url 'account_change_password' %}">{% trans "Change Password" %}</a></li>
           <li class="{% block profile-admin-change-email %}{% endblock %}"><a href="{% url 'account_email' %}">{% trans "Change Email" %}</a></li>
+          <li class="{% block profile-admin-tokens %}{% endblock %}"><a href="{% url 'profiles_tokens' %}">{% trans "Personal Access Tokens" %}</a></li>
```

Comment:
Is there a reason this is "Personal Access Tokens". I think "API Tokens" seems a bit clearer.

Reply:
No. I wasn't sure how to name it and I took the name from GitHub.

Reply:
👍 on API tokens.